### ISSUE TITLE
Multi-sig address not recognised

### ISSUE THREAD

**Issue by Cybnate** [Cybnate/NuBits-Abe-explorer#1]
It appears that multi-sig addresses are not recognised in Abe. An example is: BhCnQrYrA5LZm871dtMQEXeU93gmqbhdrC. Not sure how hard this is to fix.

See also discussion here: https://discuss.nubits.com/t/proposal-to-create-a-strategic-reserve-of-nubits/448/25

**Comment by MatthewLM** [comment]
Hi, unfortunately there are a lot of changes to merge from upstream for this. I took a look and there are many conflicts to resolve. It would probably be easier to take the latest upstream code and reimplement all of the changes to it. Not only this but the new upstream changes could introduce new problems to resolve for the functioning of the Android app and the correct working of the explorer in regards to Nubits.

Either way it's not needed for the working of the app.

**Comment by Cybnate** [comment]
Fair enough. Agree it is not required functionality for the App. Will keep it open as a known issue, if someone feels keen to work on it feel free. However I would prefer to keep the NuBits ABE explorer and Android App working together instead of creating an advanced fork which won't work with Android App.
